views/js/runner/plugins/security/fullScreen.js

```diff
@@ -225,14 +234,23 @@ define([
          */
         init: function init() {
             const testRunner = this.getTestRunner();
-            const dialogParams = {};
+            let dialogParams = {};
```

Comment:
**chore:** Why is it changed to `let`? I don't see any reassignment of the variable. `const` still fit the needs, even of the object properties are mutated, the assigned reference remains.

Reply:
Yes, you are right, my bad. 
I did an experiment with this and I forget to revert it.